diff --git a/node_modules/react-native-jitsi-meet/android/src/main/java/com/reactnativejitsimeet/RNJitsiMeetModule.java b/node_modules/react-native-jitsi-meet/android/src/main/java/com/reactnativejitsimeet/RNJitsiMeetModule.java
index e10602d..cf60114 100644
--- a/node_modules/react-native-jitsi-meet/android/src/main/java/com/reactnativejitsimeet/RNJitsiMeetModule.java
+++ b/node_modules/react-native-jitsi-meet/android/src/main/java/com/reactnativejitsimeet/RNJitsiMeetModule.java
@@ -58,7 +58,25 @@ public class RNJitsiMeetModule extends ReactContextBaseJavaModule {
                             .setAudioOnly(false)
                             .setUserInfo(_userInfo)
                             .setFeatureFlag("pip.enabled", false)
+                            .setFeatureFlag("chat.enabled", false)
                             .setFeatureFlag("calendar.enabled", false)
+                            .setFeatureFlag("call-integration.enabled",false)
+                            .setFeatureFlag("close-captions.enabled",false)
+                            .setFeatureFlag("invite.enabled",false)
+                            .setFeatureFlag("live-streaming.enabled",false)
+                            .setFeatureFlag("meeting-name.enabled",false)
+                            .setFeatureFlag("meeting-password.enabled",false)
+                            .setFeatureFlag("raise-hand.enabled",false)
+                            .setFeatureFlag("video-share.enabled",false)
+                            .setFeatureFlag("recording.enabled", false)
+                            .setFeatureFlag("help.enabled", false)
+                            .setFeatureFlag("lobby-mode.enabled", false)
+                            .setFeatureFlag("kick-out.enabled", false)
+                            .setFeatureFlag("filmstrip.enabled", false)
+                            .setFeatureFlag("tile-view.enabled", false)
+                            .setFeatureFlag("overflow-menu.enabled", false)
+                            .setFeatureFlag("toolbox.alwaysVisible", true)
+                            .setFeatureFlag("toolbox.enabled", false)
                             .build();
                     mJitsiMeetViewReference.getJitsiMeetView().join(options);
                 }
@@ -93,7 +111,25 @@ public class RNJitsiMeetModule extends ReactContextBaseJavaModule {
                             .setAudioOnly(true)
                             .setUserInfo(_userInfo)
                             .setFeatureFlag("pip.enabled", false)
+                            .setFeatureFlag("chat.enabled", false)
                             .setFeatureFlag("calendar.enabled", false)
+                            .setFeatureFlag("call-integration.enabled",false)
+                            .setFeatureFlag("close-captions.enabled",false)
+                            .setFeatureFlag("invite.enabled",false)
+                            .setFeatureFlag("live-streaming.enabled",false)
+                            .setFeatureFlag("meeting-name.enabled",false)
+                            .setFeatureFlag("meeting-password.enabled",false)
+                            .setFeatureFlag("raise-hand.enabled",false)
+                            .setFeatureFlag("video-share.enabled",false)
+                            .setFeatureFlag("recording.enabled", false)
+                            .setFeatureFlag("help.enabled", false)
+                            .setFeatureFlag("lobby-mode.enabled", false)
+                            .setFeatureFlag("kick-out.enabled", false)
+                            .setFeatureFlag("filmstrip.enabled", false)
+                            .setFeatureFlag("tile-view.enabled", false)
+                            .setFeatureFlag("overflow-menu.enabled", false)
+                            .setFeatureFlag("toolbox.alwaysVisible", true)
+                            .setFeatureFlag("toolbox.enabled", false)
                             .build();
                     mJitsiMeetViewReference.getJitsiMeetView().join(options);
                 }
diff --git a/node_modules/react-native-jitsi-meet/ios/RNJitsiMeetViewManager.m b/node_modules/react-native-jitsi-meet/ios/RNJitsiMeetViewManager.m
index c019d0a..e61bd4f 100644
--- a/node_modules/react-native-jitsi-meet/ios/RNJitsiMeetViewManager.m
+++ b/node_modules/react-native-jitsi-meet/ios/RNJitsiMeetViewManager.m
@@ -46,6 +46,7 @@ RCT_EXPORT_METHOD(call:(NSString *)urlString userInfo:(NSDictionary *)userInfo)
             builder.userInfo = _userInfo;
 
             [builder setFeatureFlag:@"pip.enabled" withBoolean:NO];
+            [builder setFeatureFlag:@"chat.enabled" withBoolean:NO];
             [builder setFeatureFlag:@"calendar.enabled" withBoolean:NO];
         }];
         [jitsiMeetView join:options];
@@ -75,6 +76,7 @@ RCT_EXPORT_METHOD(audioCall:(NSString *)urlString userInfo:(NSDictionary *)userI
             builder.audioOnly = YES;
 
             [builder setFeatureFlag:@"pip.enabled" withBoolean:NO];
+            [builder setFeatureFlag:@"chat.enabled" withBoolean:NO];
             [builder setFeatureFlag:@"calendar.enabled" withBoolean:NO];
         }];
         [jitsiMeetView join:options];
